tests/unittests/CoreGraphics.Drawing/CGPathDrawingTests.cpp

```diff
@@ -249,3 +249,192 @@ DISABLED_DRAW_TEST_F(CGPath, GetBoundingBox, UIKitMimicTest) {
+            colorComponents[0] = 0;
+            colorComponents[1] = 1;
+            colorComponents[2] = 1;
+            CGContextSetRGBStrokeColor((CGContextRef)context,
+                                       colorComponents[0],
+                                       colorComponents[1],
+                                       colorComponents[2],
+                                       colorComponents[3]);
+
+            CGContextStrokeRect((CGContextRef)context, CGRectMake(element->points[0].x - 2.5, element->points[0].y - 2.5, 4, 4));
+            CGContextStrokeRect((CGContextRef)context, CGRectMake(element->points[1].x - 2.5, element->points[1].y - 2.5, 4, 4));
+            break;
+        case kCGPathElementAddCurveToPoint:
+            colorComponents[0] = 1;
+            colorComponents[1] = 0;
+            colorComponents[2] = 1;
+            CGContextSetRGBStrokeColor((CGContextRef)context,
+                                       colorComponents[0],
+                                       colorComponents[1],
+                                       colorComponents[2],
+                                       colorComponents[3]);
+
+            CGContextStrokeRect((CGContextRef)context, CGRectMake(element->points[0].x - 2.5, element->points[0].y - 2.5, 4, 4));
+            CGContextStrokeRect((CGContextRef)context, CGRectMake(element->points[1].x - 2.5, element->points[1].y - 2.5, 4, 4));
+            CGContextStrokeRect((CGContextRef)context, CGRectMake(element->points[2].x - 2.5, element->points[2].y - 2.5, 4, 4));
+            break;
+        case kCGPathElementCloseSubpath:
+            break;
+    }
+}
+
+DISABLED_DRAW_TEST_F(CGPath, PathApplyControlPointsQuadCurve, UIKitMimicTest) {
+    CGContextRef context = GetDrawingContext();
+    CGRect bounds = GetDrawingBounds();
+    CGFloat width = bounds.size.width;
+    CGFloat height = bounds.size.height;
+    CGFloat xstart = bounds.origin.x;
+    CGFloat ystart = bounds.origin.y;
+
+    CGMutablePathRef thepath = CGPathCreateMutable();
+    CGPathMoveToPoint(thepath, NULL, xstart + .25 * width, ystart + .25 * height);
+    CGPathAddQuadCurveToPoint(thepath, NULL, xstart + .31 * width, ystart + .12 * height, xstart + .37 * width, ystart + .25 * height);
+    CGPathMoveToPoint(thepath, NULL, xstart + .5 * width, ystart + .25 * height);
+    CGPathAddQuadCurveToPoint(thepath, NULL, xstart + .56 * width, ystart + .12 * height, xstart + .62 * width, ystart + .25 * height);
+    CGPathMoveToPoint(thepath, NULL, xstart + .25 * width, ystart + .5 * height);
+    CGPathAddQuadCurveToPoint(thepath, NULL, xstart + .5 * width, ystart + .78 * height, xstart + .75 * width, ystart + .25 * height);
+    CGPathMoveToPoint(thepath, NULL, xstart + .71 * width, ystart + .26 * height);
+    CGPathAddQuadCurveToPoint(thepath, NULL, xstart + .75 * width, ystart + .22 * height, xstart + .79 * width, ystart + .27 * height);
+
+    CGContextAddPath(context, thepath);
+    CGContextStrokePath(context);
```

Comment:
Stretch goal: a test fixture that inherited UIKitMimicTest and added all of this boilerplate that drew and applied paths.